firebase-app-distribution/src/main/java/com/google/firebase/appdistribution/FirebaseAppDistribution.java

```diff
@@ -135,12 +211,71 @@ public UpdateTask updateApp() {
     return (UpdateTask) Tasks.forResult(new UpdateState(0, 0, UpdateStatus.PENDING));
   }
 
-  private boolean supportsCustomTabs(Context context) {
-    Intent customTabIntent = new Intent("android.support.customtabs.action.CustomTabsService");
-    customTabIntent.setPackage("com.android.chrome");
-    List<ResolveInfo> resolveInfos =
-        context.getPackageManager().queryIntentServices(customTabIntent, 0);
-    return resolveInfos != null && !resolveInfos.isEmpty();
+  /** Returns true if the App Distribution tester is signed in */
+  @NonNull
+  public boolean isTesterSignedIn() {
+    return false;
+  }
+
+  /** Signs out the App Distribution tester */
+  public void signOutTester() {}
+
+  @Override
+  public void onActivityCreated(
+      @NonNull @NotNull Activity activity, @androidx.annotation.Nullable @Nullable Bundle bundle) {
+    Log.d(TAG, "Created activity: " + activity.getClass().getName());
+    // if signinactivity is created, sign-in was succesful
+    if (currentlySigningIn && activity instanceof SignInResultActivity) {
```

Comment:
What is the case that currentlySigningIn is false and the activity is `instanceof` SignInResultActivity?

Reply:
Good point that should never happen, ```if (activity instanceof SignInResultActivity) ``` would be enough